### PR TITLE
Release v1.4.0

### PR DESCRIPTION
**Pull request by ashumski-adnoc**
## Purpose
Create `ui-plugin-find-contact` release
https://issues.folio.org/browse/UIPFCONT-2
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "eslint": "^5.5.0",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-no-only-tests": "^2.3.0",
+    "eslint-plugin-jest": "^23.0.4",
```
Comment (NikitaSedyx) [inline package.json:40]: same

**Review by NikitaSedyx — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Pre-Merge Checklist
+Before merging this PR, please go through the following list and take appropriate actions.
+- [ ] I've added appropriate record to the CHANGELOG.md
+- Does this PR meet or exceed the expected quality standards?
+  - [ ] Code coverage on new code is 80% or greater
+  - [ ] Duplications on new code is 3% or less
+  - [ ] There are no major code smells or security issues
+- Does this introduce breaking changes?
+  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
+  - [*] There are no breaking changes in this PR.
```
Comment (aliaksei-chumakou) [inline PULL_REQUEST_TEMPLATE.md]: It looks that I was wrong when added an asterisk here, it breaks the layout)

Comment (aliaksei-chumakou) [reply PULL_REQUEST_TEMPLATE.md]: just leave a space instead

File: package.json [hunk 3]
```diff
@@ -14,8 +14,8 @@
     "actsAs": ["plugin"],
     "pluginType": "find-contact",
     "okapiInterfaces": {
-      "organizations-storage.categories": "1.0",
-      "organizations-storage.contacts": "2.0"
+      "organizations-storage.categories": "1.1 1.2",
```
Comment (aliaksei-chumakou) [inline package.json]: please leave 1.0, there is no need to specify several minor versions

File: package.json [hunk 4]
```diff
@@ -37,6 +37,7 @@
     "chai": "^4.2.0",
     "eslint": "^5.5.0",
     "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-jest": "^23.0.4",
     "eslint-plugin-no-only-tests": "^2.3.0",
```
Comment (aliaksei-chumakou) [inline package.json:41]: maybe      `"eslint-plugin-react-hooks": "^2.1.1", ` is missed as well

**Review by aliaksei-chumakou — APPROVED** [review]

**Pull request merged**